### DISCUSSION TITLE
Toggle plugin

### DISCUSSION THREAD

**Discussion by xopo** [LazyVim/LazyVim#3767] [Q&A]
I want to use copilot for coding purpose, but I don't want it enabled when i do other text related work. I want it to be off by default and have a profile named Code  that would trigger the plugin to be usable. How can one do that? Is it something that can be done, or should a feature be build for this?
Thanks

**Comment by Saplyn** [comment]
Maybe [this](https://github.com/LazyVim/LazyVim/discussions/4232) would be helpful?

> ```lua
> -- lua/plugins/copilot.lua
> return {
>   "zbirenbaum/copilot.lua",
>   keys = {
>     {
>       "<leader>at",
>       function()
>         if require("copilot.client").is_disabled() then
>           require("copilot.command").enable()
>         else
>           require("copilot.command").disable()
>         end
>       end,
>       desc = "Toggle (Copilot)",
>     },
>   },
> }
> ```
